.github/workflows/main.yml

```diff
@@ -202,6 +202,12 @@ jobs:
           make $ch_makej
           sudo install -D -m644 argp.h /usr/include/x86_64-linux-musl/argp.h
           sudo install -D -m755 libargp.a /usr/lib/x86_64-linux-musl/libargp.a
+          # musl-gcc does not install Linux headers. This is a bad kludge.
+          # See: https://www.openwall.com/lists/musl/2017/11/23/1
```

Comment:
For lines 207-210, my thoughts:
- This looks to me like it's going to cause problems for jailed/installrooted builds.  There's probably something I'm missing since this section also calls `sudo` unconditionally, so I'll try to dig a little deeper (time permitting).
- Since only 3 of the 4 symlinks are absolute (the last one being relative), I'm assuming thought and intent have gone into making this determination already, but it's still worth a mention.  Specifically, is the intent to point to content on the root filesystem no matter what, or would the Linux headers likely be installed with the same prefix as the `musl` stuff?  You likely already know this, but just in case, `sudo ln -s ../linux ../asm-generic ../x86_64-linux-gnu /usr/include/x86_64-linux-musl/`
- This may be too nit-picky, but I would recommend placing a trailing `/` at the end of `/usr/include/x86_64-linux-musl` on all 4 lines.  Reading over these lines, it wasn't immediately obvious (to me) that the symlink path was a pre-existing directory into which new symlinks with the same names were being dropped; placing the trailing `/` at the end makes it intuitively obvious that it's a directory and not a file/symlink.  Solely cosmetic and entirely subjective, so feel free to ignore! 😀

Reply:
Indeed, I agree this is really ugly. However the only place it’s used is to set up the GitHub CI virtual machine, so I felt mostly OK about it.